upgrade/rendering-changes/4-3-0.md

```diff
@@ -21,6 +21,53 @@ position: 1
 
 The default icon type in the Telerik UI for Blazor suite changes from `FontIcon` to `SvgIcon`. This marks the next milestone in a series of improvements related to the [Content Security Policy (CSP)]({%slug troubleshooting-csp%}).
 
+### Switching to SVG icons
+
+When upgrading your application to 4.3.0 you might experience two possible scenarios related to the new default icon type:
+
+#### Changes In The Rendering
+
+The built-in icons in the Telerik UI for Blazor components will now render with different CSS classes which will break any CSS selectors that are customizing the icons:
+
+<table>
+    <thead>
+        <tr>
+            <th>UI for Blazor 4.2.0</th>
+            <th>UI for Blazor 4.3.0</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&ltspan class="k-icon k-i-camera"&gt;&lt;/span&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&ltspan class="k-svg-icon k-svg-i-camera"&gt;&lt;/span&gt;
```

Comment:
Are you sure this is all the new rendering? I think we should be rendering `<svg>` elements as well.